cloud/services/publicloadbalancers/publicloadbalancers.go

```diff
@@ -115,6 +115,7 @@ func (s *Service) Reconcile(ctx context.Context, spec interface{}) error {
 					{
 						Name: to.StringPtr("LBRuleHTTPS"),
 						LoadBalancingRulePropertiesFormat: &network.LoadBalancingRulePropertiesFormat{
+							DisableOutboundSnat:  to.BoolPtr(true),
```

Comment:
why are we doing this?

Reply:
The current configuration blocks time synchronization on the control plane Machines: https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/535

Reply:
because you can't have outbound snat enabled on your LB rules to have outbound rules ( 😢 ) see the first step in https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-outbound-rules-overview#enable-both-tcp--udp-protocols-for-outbound-nat-with-a-public-standard-load-balancer ("Disable outbound SNAT on the load balancing rule.")

Reply:
basically replacing this functionality with a dedicated outbound rule

Reply:
🤕